azure-iot-provisioning-devicesdk/azure/iot/provisioning/devicesdk/transport/mqtt_transport.py

```diff
@@ -0,0 +1,82 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import uuid
+import logging
+import six.moves.urllib as urllib
+from . import device_provisioning_constant
+from azure.iot.hub.devicesdk.transport.mqtt.mqtt_provider import MQTTProvider
+from .symmetric_key_transport import SymmetricKeyTransport
+
+logger = logging.getLogger(__name__)
+
+
+class MQTTTransport(SymmetricKeyTransport):
+    def __init__(self, provisioning_host, security_provider):
+        SymmetricKeyTransport.__init__(self, security_provider)
+        self._provisioning_host = provisioning_host
+        self._create_mqtt_provider()
+
+    def send_registration_request(self):
```

Comment:
> send_registration_request [](start = 7, length = 26)

missing docs on all methods in this file

Reply:
oh you copied it from devicesdk, which has flawed docs. I see. Not your problem. 

---
In reply to: [270205541](https://github.com/Azure/azure-iot-sdk-python-preview/pull/59#discussion_r270205541) [](ancestors = 270205541)